chalk/core.py

```diff
@@ -167,18 +154,6 @@ def atop(self, other: "Diagram") -> "Diagram":
     __add__ = atop
 
     def beside(self, other: "Diagram") -> "Diagram":
-        """Given two diagrams ``a`` and ``b``, ``a.beside(b)``
-        places ``b`` on the right side of ``a``. This moves
-        ``b`` up to touch ``a``.
-
-        💡 ``a.beside(b)`` is equivalent to ``a | b``.
-
-        Args:
-            other (Diagram): Another diagram object.
-
-        Returns:
-            Diagram: A diagram object.
```

Comment:
Same as [this comment](https://github.com/danoneata/chalk/pull/58/files#r893055676).